lib/jobs/dell-wsman-RAID.js

```diff
@@ -0,0 +1,99 @@
+// Copyright © 2017 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = DellWsmanRAIDFactory;
+di.annotate(DellWsmanRAIDFactory, new di.Provide('Job.Dell.Wsman.RAID'));
+di.annotate(DellWsmanRAIDFactory, new di.Inject(
+    'Job.Dell.Wsman.Base',
+    'JobUtils.WsmanTool',
+    'Logger',
+    'Assert',
+    'Util',
+    'Services.Configuration',
+    'Errors',
+    'Services.Encryption',
+    'validator'
+));
+
+function DellWsmanRAIDFactory(
```

Comment:
why no `_run` callback?

Reply:
_run callback is defined in it's parent class